modules/highgui/src/window_cocoa.mm

```diff
@@ -849,17 +765,17 @@ void cvSetPropTopmost_COCOA( const char* name, const bool topmost )
         EXIT;
     }
 
-    localpool = [[NSAutoreleasePool alloc] init];
-    if (topmost)
-    {
-        [window makeKeyAndOrderFront:window.self];
-        [window setLevel:CGWindowLevelForKey(kCGMaximumWindowLevelKey)];
-    }
-    else
-    {
-        [window makeKeyAndOrderFront:nil];
+    @autoreleasepool{
```

Comment:
I think `@autoreleasepool` block should include `CVWindow* window = cvGetWindow(name);` as it is done in other functions